spec/controllers/replicants_controller_spec.rb

```diff
@@ -28,15 +28,4 @@
       end
     end
   end
-
-  GOOD_REPLICANTS.each do |name, ua_string|
```

Comment:
There's no such thing as `GOOD_REPLICANTS` in this library now. This is probably leftover from the original version of voight_kampff

Reply:
Actually, this is my bad. I had added this but forgot to ever check in: `spec/support/good_replicants.rb`

Here is the content if you want to add it in this PR rather than removing this.
```
GOOD_REPLICANTS = {
  'Wget' => 'Wget/1.20.3 (darwin19.0.0)',
  'Curl' => 'libcurl/7.19.7 r-curl/3.1 httr/1.3.1'
}
```

Reply:
Oh nice! Thanks for the context. I'll put that back in